src/gamesave.lua

```diff
@@ -26,7 +26,7 @@ function Gamesave.save(saveName, world)
 
 	local fileContents = SceneParser.saveScene(world)
 
-	if not love.filesystem.isDirectory(saveDir) then
+	if not love.filesystem.getInfo(saveDir) then
```

Comment:
You can verify that the save directory really is a directory in one command by passing a second argument (filtertype) to getInfo. See https://love2d.org/wiki/love.filesystem.getInfo